### DISCUSSION TITLE
Near-term vetting: Carbon Management

### DISCUSSION THREAD

**Discussion by RahelMA** [pik-piam/mrremind#544] [Near-term validation]
**Variable: `Carbon Capture|Storage`**

**Source: IEA-CCUS-Database 2024**

* country data
* available project status: "operational", "under construction", "planned", ("decommissioned", "suspended")

Assumptions + data preparation:

* slow technology: potential new power plants that are not yet under construction won't be operational in 2030
* if decommissioning date empty, assume no decommissioning before 2030
* remove entries without announced capacities or start date
*  remove data of country "Unknown"
* use project types: "Full chain", "Storage", "T&S"
* no projects are under way in Brasil which means all thresholds are equal to avoid this, add manually an upper estimate of what could still be planned in BRA
* projects shared between multiple countries are allocated according to GDP

Threshold definitions:

**2020**
* global  `min_red`: -10% ("operational") ; regional  `min_red`: -40% ("operational")
* global `min_yel`:  -5% ("operational")   ; regional  `min_yel`: -20% ("operational")
* global `max_yel`: +5% ("operational")  ; regional  `max_yel`: +20% ("operational")
* global `max_red`: +10% ("operational") ; regional  `max_red`: +40% ("operational")

**2025**
* global  `min_red`: -10% ("operational" * 0.9) 
* global `min_yel`:  -5% ("operational" + "construction" * 0.2)     ; regional  `min_yel`: -40% ("operational" + "construction" * 0.2) 
* global `max_yel`: +5% ("operational" + "construction" +"planned" * 0.2)   ; regional  `max_yel`: +40% ("operational" + "construction" +"planned" * 0.2) 
* global `max_red`: +10%  ("operational" + "construction" +"planned") 

**2030**
* global  `min_red`: -10% ("operational") 
* global `min_yel`:  -5% ("operational" + "construction" * 0.5)     ; regional  `min_yel`: -40% ("operational" + "construction" * 0.5) 
* global `max_yel`: +5% ("operational" + "construction" +"planned" * 0.2)   ; regional  `max_yel`: +40% ("operational" + "construction" +"planned" * 0.2) 
* global `max_red`: +10%  ("operational" + "construction" +"planned") 

Data Source:
https://www.iea.org/data-and-statistics/data-product/ccus-projects-database (15.03.2024)

ChangeLog:
- update data source: using IEA CCUS data from 30.04.2025
- update data source: using IEA CCUS data from 27.03.2026

**Comment by EHooijschuur** [comment]
The lower red boundary on carbon management includes technologies that we do not model, such as CCU projects on natural gas and oil extraction. 

Additionally, for the 2030 max yellow value, we are not sure whether it makes sense to assume that only 20% of planned projects will be actually implemented. If the higher yellow boundary should include values that are likely, shouldn’t we trust that at least the planned projects will happen? Or are these projects planned to be finished after 2030?
